index.js

```diff
@@ -0,0 +1,111 @@
+const playerContainer = document.querySelector(".player-container")
+
+function renderPlayer(player) {
+  const playerDiv = document.createElement("div")
+  playerDiv.className = "player"
+  playerDiv.setAttribute("player-id", player.id)
+  playerDiv.dataset.number = player.number
+  playerDiv.innerHTML = `
+    <h3>${player.name} (<em>${player.nickname}</em>)</h3>
+    <img src="${player.photo}" alt="${player.name}">
+    <p>Jersey #${player.number}</p>
+    <p class="likes">${player.likes} likes</p>
+    <button class="like-button">❤️</button>
+    `
+  playerContainer.append(playerDiv)
+
+  //******DELIVERABLE #3 ************************
+  const likeButton = playerDiv.querySelector("button.like-button")
+  const showLikes = playerDiv.querySelector("p.likes")
+  likeButton.addEventListener("click", increaseLikes)
+
+  function increaseLikes() {
+  player.likes++ ;
+    
+  let configObject = {
+    method: "PATCH",
+    headers: {
+      "Content-Type": "application/json",
+      "Accept": "application/json",
+    },
+      body: JSON.stringify({ likes: player.likes })
+  }
+    return fetch("http://localhost:3000/players" + `/${player.id}`, configObject)
+      .then(function(response){
+        return response.json()
+    }).then(function(data){
+        showLikes.textContent = `${player.likes} likes.`
+    }) .catch(function(error) {
+        alert(error.message)
+    })
+  }
```

Comment:
Nice use of a nested callback function here to give yourself access to some additional data 👍 

Just be careful about your indentation, it can be harder to spot syntax errors/harder to keep track of variables in nested functions when your code isn't indented well.